.codeclimate.yml

```diff
@@ -0,0 +1,16 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  eslint:
+    enabled: true
+ratings:
+  paths:
+  - "**.js"
+  - "**.jsx"
```

Comment:
Yeah, some of this stuff was auto-generated by codeclimate...i paired it down, but we don't need this.